Ji4n1ng/OpenInTerminal
# Add support for Ghostty.

## Summary of this pull request

Add support for [Ghostty](https://ghostty.org) terminal.

## Does this solve an existing issue? If so, add a link to it

- #224

## Steps to test this feature

- Install [Ghostty](https://ghostty.org).
- Build OpenInTerminal from sources.
- Check if Ghostty is available within OpenInTerminal available apps list.

## Screenshots

-

## Additional info

-
